### PR TITLE
Refactor fns and vars permissions in game & player

### PR DESCRIPTION
**Pull request by chavitos**
Refactor some variables permissions, and muts, and move some fns to the respective owners.

### CODE REVIEW

File: truco-careca-server/src/game.rs [hunk 1]
```diff
@@ -38,19 +37,25 @@ impl Game {
         self.list_players.len() == 4
     }
 
-    pub fn get_players(&mut self) -> &HashMap<u32, (Connection, Player)> {
-        &self.list_players
```
Comment (marguera) [inline truco-careca-server/src/game.rs:42]: `list_players` is not public, we prob gonna need this method some time in the future

**Review by marguera — APPROVED** [review]

File: truco-careca-server/src/game.rs (hunk 1 above)
Comment (chavitos) [reply truco-careca-server/src/game.rs:42]: @marguera  But any kind of update needed, or info, can be provided/performed by a specific fn in game or player. What do you think?

Comment (marguera) [reply truco-careca-server/src/game.rs:42]: I do agree that if we don't need that for now we delete it, I was using this to print to std out in the main.rs

Comment (marguera) [reply truco-careca-server/src/game.rs:42]: just merge this!

File: truco-careca-server/src/game/player.rs [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
-}
\ No newline at end of file
+    pub fn new(name: String, is_ready: bool) -> Self {
+        Self { name, is_ready }
+    }
+
+    pub fn update(&mut self, name: String, is_ready: bool) {
+        self.name = name;
+        self.is_ready = is_ready;
+    }
```
Comment (marguera) [inline truco-careca-server/src/game/player.rs:15]: 😍

**Pull request merged**